core/jobs.py

```diff
@@ -127,11 +127,13 @@ def _create_new_job():
         return transaction_services.run_in_transaction(_create_new_job)
 
     @classmethod
-    def enqueue(cls, job_id, additional_job_params=None):
+    def enqueue(cls, job_id, queue_name='default', additional_job_params=None):
         """Marks a job as queued and adds it to a queue for processing.
 
         Args:
             job_id: str. The ID of the job to enqueue.
+            queue_name: str. The queue name the job should be run in.
+                Corresponds to the values in queue.yaml.
```

Comment:
Perhaps refer to taskqueue_services instead -- and only have the reference to queue.yaml in taskqueue_services itself. The rationale here is that this is the logic layer, and it shouldn't really need to know the platform-specific details.